generators/server/templates/src/test/java/package/web/rest/AccountResourceIT_oauth2.java.ejs

```diff
@@ -39,7 +30,7 @@ import <%= packageName %>.service.UserService;
 import org.junit.jupiter.api.BeforeEach;
 <%_ } _%>
 import org.junit.jupiter.api.Test;
-<%_ if (databaseType === 'neo4j' || cacheProvider === 'redis' || (reactive && ['mysql', 'postgresql', 'mssql'].includes(prodDatabaseType))) { _%>
+<%_ if (reactive && ['mysql', 'postgresql', 'mssql'].includes(prodDatabaseType)) { _%>
```

Comment:
I believe this can be removed too, this condition is the same as 'reactiveSqlTestContainers'

Reply:
Agreed, its an unused import.